pkg/cosign/tuf/client_test.go

```diff
@@ -51,12 +51,7 @@ func TestNewFromEnv(t *testing.T) {
 	tuf.Close()
 	checkTargetsAndMeta(t, tuf)
 
-	// Now let's explicitly make a root.
-	remote, err := GcsRemoteStore(ctx, DefaultRemoteRoot, nil, nil)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if err := Initialize(remote, nil); err != nil {
+	if err := Initialize(ctx, DefaultRemoteRoot, nil); err != nil {
```

Comment:
Can we update the test to check that we've written remote.txt?

Reply:
yeah! actually gonna add a new test of custom root, that would help some later on tests anyway

Reply:
added a test that actually uses the cached remote.txt location to pull more metadata!